linebot/account_link_test.go

```diff
@@ -54,7 +64,8 @@ func TestIssueLinkToken(t *testing.T) {
 		if r.Method != http.MethodPost {
 			t.Errorf("Method %s; want %s", r.Method, http.MethodPost)
 		}
-		endpoint := fmt.Sprintf(APIEndpointLinkToken, tc.UserID)
+		// if path variable is empty, "//" will be normalized in request
```

Comment:
We should fix `IssueLinkTokenCall.Do` so that  if `UserID` is empty, `IssueLinkTokenCall.Do` should return an error without API call.

```go
func (call *IssueLinkTokenCall) Do() (*LinkTokenResponse, error) {
	if call.userID == "" {
		return nil, fmt.Errorf("userID is required")
	}
	endpoint := fmt.Sprintf(APIEndpointLinkToken, call.userID)
	res, err := call.c.post(call.ctx, endpoint, nil)
	if err != nil {
		return nil, err
	}
	defer closeResponse(res)
	return decodeToLinkTokenResponse(res)
}
```

Reply:
yes, i agree with you but it should be done in another PR

Reply:
I agree. I create the issue and resolve this conversaiton.
Thanks.
https://github.com/line/line-bot-sdk-go/issues/163